extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -1079,8 +1079,10 @@ public String getSupportInfo() {
 
             // Search for correct panel containing the data
             for (int i = 0; i < panels.size(); i++) {
-                if (panels.getObject(i).getObject("engagementPanelSectionListRenderer")
-                        .getString("panelIdentifier").equals("engagement-panel-macro-markers")) {
+                final String panelIdentifier = panels.getObject("engagementPanelSectionListRenderer")
+                        .getString("panelIdentifier")
```

Comment:
```suggestion
                        .getString("panelIdentifier");
```

Reply:
Sorry, I didn't use Android Studio for this but Notepad++ so I don't saw this :(.